### PR TITLE
Sets up Basic Routing

### LINKED ISSUE #11: Router

### PR DESCRIPTION
**Pull request by skmshaffer**
Adds Basic Routing to the Application, sets up some basic pages as examples including an example of path variables, adds a 404 Not Found Page to the application. 

Closes #11.

### CODE REVIEW

File: src/Organization.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    } else {
+        component = <Two/>
+    }
+
+    return(
+        <div>
+            { component }
+        </div>
+    );
+}
```
Comment (jSith) [inline src/Organization.js]: Forgot a semicolon 😮

File: src/Organization.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from "react"; 
+import "./Organization.css"; 
+import { Route } from "react-router-dom";
```
Comment (jSith) [inline src/Organization.js]: unused import

File: src/NotFound.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                        Uh oh! The page you were looking for wasn't found. 
+                        We're not quite sure how you got here, but you did. 
+                        So please try again.
+                    </p>
+                </Col>
+            </Row>
+            <Row>
+                <Col md="12">
+                    <div class="text-center">
+                    <img className="rounded" src="https://media.giphy.com/media/EsmlrgWNx5v0Y/giphy.gif"/>
```
Comment (jSith) [inline src/NotFound.js]: I think you're also supposed to pass an "alt" text prop for any img. Not that it really matters here lol

Also nice Obama

File: src/Nav.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React from "react"; 
+import './Nav.css';
+
+import {
+    Link, 
+} from "react-router-dom";
+
+import {
+    Navbar,
+    NavbarBrand,
```
Comment (jSith) [inline src/Nav.js]: unused import

(for now, tho we will probably want to add a brand at some point)

File: src/Nav.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+} from "reactstrap";
+
+
+const Nav = (props) => {
```
Comment (jSith) [inline src/Nav.js]: Prefer () since you don't actually use props

File: src/Nav.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    return(
+    <div>
+        <Navbar color="light">
+            <span className="navbar-brand">
+                <Link className="brand-text" to="/">NVolve2</Link>
+            </span>
+        </Navbar>
+    </div>);
```
Comment (jSith) [inline src/Nav.js]: Put this close paren on her own line please. She is a strong independent woman and she don't need no div.

File: src/App.js [hunk 7]
```diff
@@ -1,25 +1,27 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Nav />
+      <Switch>
+        <Route path="/organizations/:org" component={Organization}></Route>
```
Comment (jSith) [inline src/App.js]: You should be able to close this with just /> since it doesn't actually contain anything

File: src/App.js [hunk 8]
```diff
@@ -1,25 +1,27 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router, 
+  Switch, 
+  Route, 
+  Link
```
Comment (jSith) [inline src/App.js]: unused import :o

File: src/App.js [hunk 9]
```diff
@@ -1,25 +1,27 @@
 import React from 'react';
 import logo from './logo.svg';
```
Comment (jSith) [inline src/App.js]: unused import

File: src/Organization.js [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+    let { org } = props.match.params;
+    let component; 
+
+    if (org === "test") {
+        component = <One/>
+    } else {
+        component = <Two/>
+    }
+
+    return(
```
Comment (jSith) [inline src/Organization.js]: What about instead: 
```suggestion
return (
   org === "test" ? <One /> : <Two />
);
```
🤷‍♀

File: src/Organization.js [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+        <div>
+            <h1>Two!</h1>
+        </div>
+    );
+};
+
+const Organization = (props) => {
+
+    let { org } = props.match.params;
```
Comment (jSith) [inline src/Organization.js]: letconst

**Review by jSith — CHANGES_REQUESTED** [review]

**Comment by jSith** [comment]
Nice work!

File: src/Organization.js (hunk 1 above)
Comment (skmshaffer) [reply src/Organization.js]: fixed.

File: src/Organization.js (hunk 2 above)
Comment (skmshaffer) [reply src/Organization.js]: Fixed.

File: src/Organization.js (hunk 11 above)
Comment (skmshaffer) [reply src/Organization.js]: Done. As you can see, Karl does not write JavaScript. Karl does not know Javascript.

File: src/NotFound.js (hunk 3 above)
Comment (skmshaffer) [reply src/NotFound.js]: Done.

File: src/Nav.js (hunk 4 above)
Comment (skmshaffer) [reply src/Nav.js]: Done.

File: src/Nav.js (hunk 5 above)
Comment (skmshaffer) [reply src/Nav.js]: Done.

File: src/App.js (hunk 7 above)
Comment (skmshaffer) [reply src/App.js]: Done

File: src/App.js (hunk 8 above)
Comment (skmshaffer) [reply src/App.js]: Done.

File: src/App.js (hunk 9 above)
Comment (skmshaffer) [reply src/App.js]: Done

**Comment by skmshaffer** [comment]
@jSith Changes made. Should be good to go now. I also added and used prettier to reformat all of my things.

**Review by jSith — APPROVED** [review]
Looks good!

**Review by gauriramesh — CHANGES_REQUESTED** [review]
Can you please move all of the components to a "components" folder so we don't clutter up the src/ folder? Such as dashboard and nav, etc.

**Comment by skmshaffer** [comment]
@gauriramesh @fitzmill @jSith Donzo! Organized created components into ./pages/ and ./components/ directories.

**Review by gauriramesh — APPROVED** [review]
looks good!

**Review by fitzmill — APPROVED** [review]

**Pull request merged**